plugins/cypress/src/tasks/cypress.ts

```diff
@@ -15,7 +15,7 @@ export class CypressLocal extends Task<typeof CypressSchema> {
     const vault = new DopplerEnvVars(this.logger, 'dev')
     const vaultEnv = await vault.get()
 
-    const env = { ...process.env, ...cypressEnv, ...vaultEnv }
+    const env = { ...process.env, ...vaultEnv, ...cypressEnv }
```

Comment:
**suggestion**: can we rename these vault variables to denote doppler instead whilst we're here?